feat(webapp): changed loading bpjson data

AngeloCG97 opened the pull request:
### Changed loading bpjson data

xavier506 committed:
fix(webapp): update mailchimp

xavier506 committed:
fix(webapp): social media preview

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

xavier506 committed:
Merge branch 'dev'

AngeloCG97 committed:
feat(webapp): changed loading bpjson data

AngeloCG97 committed:
feat(webapp): updated content homepage

xavier506 commented on .github/workflows/push-dev-environment.yaml:
```diff
@@ -40,6 +40,7 @@ jobs:
           REACT_APP_UAL_APP_NAME: 'my-vote-eos-proxy'
           REACT_APP_UAL_API_PROTOCOL: 'https'
           REACT_APP_UAL_API_HOST: 'api.jungle3.alohaeos.com'
+          REACT_APP_PRODUCERS_INFO_API_URL: 'https://graphql-eos.antelope.tools/api/rest/get-producers-info'
```
@AngeloCG97 
can we set this to use the jungle API? the jungle API should be deployed now so we can test

AngeloCG97 committed:
feat(worksflows): changed endpoint url dev environment

AngeloCG97 committed:
A

AAAAAAAAAAAAAAAAAAAAAABBBBBBBBBBBBBBBBBBBBBBBMerge branch 'dev' of https://github.com/eoscostarica/MyvoteEOS-proxy into fix/homepage

AngeloCG97 committed:
Merge pull request #139 from eoscostarica/dev

Production Release

AngeloCG97 committed:
BB
Merge branch 'master' of https://github.com/eoscostarica/MyvoteEOS-proxy into fix/homepage

AngeloCG97 committed:
feat(webapp): changed form to get bpjson data

xavier506 reviewed (approved):
LGTM
